### ISSUE TITLE
IOException in tonieHandler - Issue: "Unrecognized field "ownerName""

### ISSUE THREAD

**Issue by erikp152** [maximilianvoss/spotify-toniebox-sync#2]
IOException after starting the daemon in spotify-toniebox-sync.log:
```
. 2019 16:53:08,197 - ERROR:rocks.voss.spotifytonieboxsync.Application:main:71: IOException
com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "ownerName" (class rocks.voss.toniebox.beans.toniebox.Household), not marked as ignorable (6 known properties: , "access", "canLeave", "id", "image", "name", "foreignCreativeTonieContent"])
 at [Source: N/A; line: -1, column: -1] (through reference chain: rocks.voss.toniebox.beans.toniebox.Household["ownerName"])
	at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:79)
	at com.fasterxml.jackson.databind.DeserializationContext.reportUnknownProperty(DeserializationContext.java:555)
	at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:708)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1160)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:315)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:121)
	at com.fasterxml.jackson.databind.deser.std.ObjectArrayDeserializer.deserialize(ObjectArrayDeserializer.java:151)
	at com.fasterxml.jackson.databind.deser.std.ObjectArrayDeserializer.deserialize(ObjectArrayDeserializer.java:18)
	at com.fasterxml.jackson.databind.ObjectMapper._readValue(ObjectMapper.java:2860)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:1569)
	at com.fasterxml.jackson.databind.ObjectMapper.treeToValue(ObjectMapper.java:1902)
	at rocks.voss.toniebox.beans.Transformer.createBean(Transformer.java:23)
	at rocks.voss.toniebox.beans.Transformer.createBean(Transformer.java:16)
	at rocks.voss.toniebox.RequestHandler.executeRequest(RequestHandler.java:149)
	at rocks.voss.toniebox.RequestHandler.executeGetRequest(RequestHandler.java:106)
	at rocks.voss.toniebox.RequestHandler.getHouseholds(RequestHandler.java:55)
	at rocks.voss.toniebox.TonieHandler.getHouseholds(TonieHandler.java:55)
	at rocks.voss.spotifytonieboxsync.Application.main(Application.java:62)
```

The problem seems to be in the function "tonieHandler.getHouseholds();" of the Application.java. Propable Tonies has changed its api? Or is sth additional needed as the "https://github.com/maximilianvoss/toniebox-api" what is not described in the readme?

**Comment by udovic2100** [comment]
Having the same issue. @maximilianvoss could you comment if it's a configuration error or really something with the toniebox api that has changed? Thanks for your help!

**Comment by maximilianvoss** [comment]
Thanks Jonas for providing the fix. I released version 2.5 of tonie api (which included additional fixed on the api)
